### PR TITLE
Review

### PR DESCRIPTION
**Pull request by SanMasood**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,5 +1,198 @@
...
+  if (Number.isNaN(diff)) {
+    res.status(400).json({ error: 'Parameters must be valid numbers.' });
+  } else {
+    res.status(200).json({ result: diff });
+  }
+});
+//--------------------------------------------------
+app.post('/numbers/multiply', (req, res) => {
+  // const arr = Object.values(req.body);
+  const num1 = parseInt(req.body.a);
```
Comment (ersel) [inline src/app.js:73]: put console.logs here

```
console.log({ num1, num2 });
```

Comment (SanMasood) [reply src/app.js:73]:
```
console.log src/app.js:75
      { num1: 10, num2: 3 }
    console.log src/app.js:75
      { num1: -4, num2: -9 }
    console.log src/app.js:75
      { num1: NaN, num2: NaN }
    console.log src/app.js:75
      { num1: NaN, num2: NaN }
```

Comment (SanMasood) [reply src/app.js:73]: `  console.log src/app.js:75
      { num1: 10, num2: 3 }
    console.log src/app.js:75
      { num1: -4, num2: -9 }
    console.log src/app.js:75
      { num1: NaN, num2: NaN }
    console.log src/app.js:75
      { num1: NaN, num2: NaN }`

But this is multiply so it works fine anway. Shall I do the same for divide

Comment (ersel) [reply src/app.js:73]: yes please

Comment (SanMasood) [reply src/app.js:73]:
```

    console.log src/app.js:110
      { div: 54 }
    console.log src/app.js:110
      { div: -0.5 }
    console.log src/app.js:110
      { div: 0 }
    console.log src/app.js:110
      { div: Infinity }
    console.log src/app.js:110
      { div: NaN }
    console.log src/app.js:110
      { div: NaN }

  ● /numbers › POST /divide › divides two 
numbers

    : Timeout - Async callback was not inv
oked within the 5000ms timeout specified b
y jest.setTimeout.Timeout - Async callback
 was not invoked within the 5000ms timeout specified by jest.setTimeout.Error:

```

Comment (SanMasood) [reply src/app.js:73]: fails for 4 tests and also takes ages to run (timeout).

Comment (ersel) [reply src/app.js:73]: I want to see a and b, not div...

**Pull request closed without merging**